src/amo/components/InstallButtonWrapper/styles.scss

```diff
@@ -9,7 +9,7 @@
 }
 
 .InstallButtonWrapper-downloadLink {
-  margin-top: 12px;
+  margin: 12px auto;
```

Comment:
Any reason to use `auto` rather than `0`?

Reply:
Nope!